doc/classes/EditorInterface.xml

```diff
@@ -285,6 +285,47 @@
 				See also [method Window.set_unparent_when_invisible].
 			</description>
 		</method>
+		<method name="popup_dialog_property_selector">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<param index="1" name="callback" type="Callable" />
+			<param index="2" name="type_filter" type="PackedInt32Array" default="PackedInt32Array()" />
+			<description>
+				Pops up an editor dialog for selecting properties from [param object]. The [param callback] must take a [NodePath]. It is [method Callable.call_deferred] on the selected property path (see [method NodePath.get_as_property_path]) or the empty path [code]^""[/code] if the dialog is canceled. If [param type_filter] is provided, the dialog will only show properties that match one of the listed [enum Variant.Type] values.
+				[b]Example:[/b]
+				[codeblock]
+				func _ready() -&gt; void:
+				    if Engine.is_editor_hint():
+				        EditorInterface.popup_dialog_property_selector(this, _on_property_selected, [TYPE_INT])
+
+				func _on_property_selected(property_path : NodePath) -&gt; void:
+				    if property_path == ^"":
```

Comment:
```suggestion
				func _on_property_selected(property_path):
				    if property_path == "":
```
I think we usually avoid using static typing in the examples (though double-check with @godotengine/documentation before applying this change).

Reply:
I'd say to remove them as we indeed don't use them as a rule, don't think I've seen any exceptions in fact